### ISSUE TITLE
Uncaught TypeError: Cannot read property 'html' of undefined

### ISSUE THREAD

**Issue by radarsu** [leny/atom-w3c-validation#13]
[Enter steps to reproduce below:]
1. Saving file
2. ...

**Atom Version**: 1.0.15
**System**: Unknown Windows Version
**Thrown From**: [w3c-validation](https://github.com/leny/atom-w3c-validation) package, v0.2.0
### Stack Trace

Uncaught TypeError: Cannot read property 'html' of undefined

```
At events.js:141

TypeError: Cannot read property 'html' of undefined
  at C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\lib\w3cjs.js:105:29
  at Parser.<anonymous> (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\lib\xml2js.js:466:18)
  at emitOne (events.js:77:13)
  at Parser.emit (events.js:169:7)
  at Object.onerror (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\lib\xml2js.js:295:26)
  at emit (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\node_modules\sax\lib\sax.js:625:33)
  at error (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\node_modules\sax\lib\sax.js:654:3)
  at strictFail (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\node_modules\sax\lib\sax.js:675:22)
  at closeTag (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\node_modules\sax\lib\sax.js:858:7)
  at Object.write (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\node_modules\sax\lib\sax.js:1314:29)
  at Parser.exports.Parser.Parser.parseString (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\lib\xml2js.js:481:31)
  at Parser.parseString (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\lib\xml2js.js:7:59)
  at Object.exports.parseString (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\xml2js\lib\xml2js.js:513:19)
  at parseCssValidation (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\lib\w3cjs.js:103:23)
  at C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\lib\w3cjs.js:81:29
  at Request.callback (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\superagent\lib\node\index.js:630:3)
  at Request.<anonymous> (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\superagent\lib\node\index.js:131:10)
  at emitOne (events.js:77:13)
  at Request.emit (events.js:169:7)
  at IncomingMessage.<anonymous> (C:\Users\Artur\.atom\packages\w3c-validation\node_modules\w3cvalidator\node_modules\superagent\lib\node\index.js:773:12)
  at emitNone (events.js:72:20)
  at IncomingMessage.emit (events.js:166:7)
  at endReadableNT (_stream_readable.js:889:12)
  at doNTCallback2 (node.js:438:9)
  at process._tickCallback (node.js:352:17)

```
### Commands

```
     -0:26.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:24.7.0 core:cut (atom-text-editor.editor.is-focused)
  2x -0:24.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:20.8.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -0:19.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:17.1.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:17.1.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:17 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:17 editor:newline (atom-text-editor.editor.is-focused)
     -0:16.3.0 core:undo (atom-text-editor.editor.is-focused)
     -0:15.3.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:15.3.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:15.2.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:15.2.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:14.7.0 core:paste (atom-text-editor.editor.is-focused)
     -0:13.2.0 core:save (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "slack-chat"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
angularjs, v0.3.0
angularjs-snippets, v0.10.18
angularjs-styleguide-snippets, v0.7.2
atom-beautify, v0.28.12
atom-html-preview, v0.1.11
auto-id-class, v0.2.6
autocomplete-modules, v1.1.0
block-comment-lines, v0.2.4
clipboard-history, v0.6.6
color-picker, v2.0.12
color-tabs, v0.1.8
color-tabs-regex, v0.2.2
docblockr, v0.7.3
docs-snippets, v0.8.0
file-icons, v1.6.9
font-viewer, v0.3.0
fonts, v0.6.0
foundation5-snippets, v0.2.2
htmlhint, v1.1.3
js-patterns-atom-snippets, v0.2.1
jshint, v1.8.0
keyboard-localization, v1.4.8
keycodes, v0.1.4
local-server-express, v0.3.1
minimap, v4.13.3
pigments, v0.15.0
Remote-FTP, v0.7.2
seti-icons, v0.4.5
smart-tags, v0.1.0
space-underscore, v0.1.1
toggle-quotes, v0.11.3
trailing-spaces, v0.3.2
travis-ci-status, v0.18.0
turbo-javascript, v1.1.5
w3c-validation, v0.2.0

# Dev
No dev packages
```

**Comment by leny** [comment]
Hi, thanks for the report.
For now, I'm quite busy with some pro work.
I will not be able to review this until january. Feel free to investigate and submit a Pull Request if you find something!

**Comment by leny** [comment]
Hi, sorry for the _long_ time on this one, was very busy with my work.

I've just released a new version (`0.3.0`) of the package, using new modules for validation.

Hope it will fix this issue.
